src/Increment.js

```diff
@@ -0,0 +1,5 @@
+const Increment = (props) => {
+  return <button> onClick={props.changeScore}</button>;
```

Comment:
I see you're using props here but I don't see where you're passing the prop from.